skiko/src/awtMain/kotlin/org/jetbrains/skiko/SkiaLayer.awt.kt

```diff
@@ -53,6 +54,7 @@ actual open class SkiaLayer internal constructor(
         isVsyncEnabled: Boolean = SkikoProperties.vsyncEnabled,
         isVsyncFramelimitFallbackEnabled: Boolean = SkikoProperties.vsyncFramelimitFallbackEnabled,
         renderApi: GraphicsApi = SkikoProperties.renderApi,
+        pixelGeometry: PixelGeometry = PixelGeometry.UNKNOWN,
```

Comment:
Let's move `pixelGeometry` below `analytics` to preserve backward compatibility. Technically we still don't need to do that, but it is better to start.